.circleci/config.yml

```diff
@@ -153,6 +153,58 @@ jobs:
             terraform destroy -var project=${CLOUDSDK_CORE_PROJECT} -auto-approve
           when: always
 
+  acceptance-openshift:
+    environment:
+      - TEST_RESULTS: /tmp/test-results
+      - OC_PRIMARY_NAME: consul-helm-test-2757871175
+      - OC_SECONDARY_NAME: consul-helm-test-3737660519
```

Comment:
What are these? Can we add a comment

Reply:
These are the pre-created openshift cluster names. Will add!